src/main/java/fruitshop/model/FruitTransaction.java

```diff
@@ -0,0 +1,69 @@
+package fruitshop.model;
+
+public class FruitTransaction {
```

Comment:
Don't begin class implementation with an empty line. Remove the redundant empty line at the start of the class definition.